scripts/install.tpl.sh

```diff
@@ -38,5 +38,8 @@ kubectl_wait "cert-manager" "deployment/cert-manager-webhook" "60s"
 kubectl apply -f "https://github.com/operator-framework/catalogd/releases/download/${catalogd_version}/catalogd.yaml"
 kubectl_wait "olmv1-system" "deployment/catalogd-controller-manager" "60s"
 
+kubectl apply -f "https://raw.githubusercontent.com/operator-framework/operator-controller/main/config/samples/catalogd_operatorcatalog.yaml"
```

Comment:
I'm not sure that we should utilize the samples here as the default catalogs, despite this being one that utilizes the OperatorHub catalog. 

What do you think about adding officially maintained defaults to catalogd? This could take the form of plain YAML or another kustomize overlay, but I think the benefits of this is that the default `ClusterCatalog` resources are co-located with the code that would be modified when changes are made to the API.

Reply:
That sounds like a good idea to me, I suggested something like that in the description of the PR. I can close this PR and make those changes there. Are you thinking something where the catalog is installed by default when Catalogd is installed, and then operator-controller would just inherit that when it installs Catalogd?